app/views/owners/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<% @title = @rubygem.name %>
+
+<% content_for :head do %>
+  <meta name="robots" content="noindex"/>
+<% end %>
+
+
+<div class="row">
+  <div class="column l-column">
+
+    <div class="owners__navigation">
+      <%= link_to "← Back", rubygem_path(@rubygem), class: "owners__back__navigation" %>
```

Comment:
Could it be possible to reuse t--link-arrow here?
![Screenshot from 2020-07-09 13-32-14](https://user-images.githubusercontent.com/7680662/87013816-d64a7a00-c1e8-11ea-9b1d-1a29bea91e56.png)